src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -63,12 +63,10 @@ public boolean equals(Object other) {
         if (other == this) {
             return true;
         }
-
         // instanceof handles nulls
         if (!(other instanceof PersonCard)) {
             return false;
         }
-
```

Comment:
add line back

Reply:
adding these lines back cause checkstyle to fail somehow?